db/migrate/20210620025737_add_post_edits_count_to_user_stats.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AddPostEditsCountToUserStats < ActiveRecord::Migration[6.1]
+  def change
+    add_column :user_stats, :post_edits_count, :integer, default: 0, null: false
+
+    execute <<~SQL
+      UPDATE user_stats us
```

Comment:
I am slightly suspicious of the performance of this query in a migration. Have you been able to test with on a database with many users?

You should be able to populate your development database using our `discourse_dev` rake tools (they should be already been included in your install.) 

`rake users:populate`

I'd suggest setting up a test with many posts and users. I'm not sure if our populate tasks support edits, but if not perhaps it's something you could add?

At the least we should probably add an index on `user_id` on post_revisions, but I am curious how the performance goes otherwise.